package cn.itcast.day14.demo03Exception;

import java.util.List;

/**
 * @author yucheno8
 * @date 2021/10/8/008
 **/
public class Demo01Exception {

    public static void main(String[] args) {
        /*
            多个异常使用捕获又该如何处理呢？
                1.多个异常分别处理。
                2.多个异常一次捕获，多次处理。
                3.多个异常一次捕获一次处理。
         */
        int[] arr = {1, 2, 3};
//        System.out.println(arr[3]); // java.lang.ArrayIndexOutOfBoundsException: 3


    }
}
